### PR TITLE
Update Changelog and Release Notes for OC12

### PR DESCRIPTION
**Pull request by Rillke**
The release notes for Opencast 12.

Guide:
https://docs.opencast.org/develop/developer/#release-manager/#after-the-election-of-the-release-manager

### Your pull request should…

* [x] have a concise title
*  [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
*  include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: docs/guides/admin/docs/changelog.md [hunk 1]
```diff
@@ -218,24 +351,14 @@ Opencast 11
   Replace native javascript with angularJS code
 - [[#3127](https://github.com/opencast/opencast/pull/3127)] -
   Fix minor bugs in AdaptivePlaylist
-- [[#3180](https://github.com/opencast/opencast/pull/3180)] -
```
Comment (Rillke) [inline docs/guides/admin/docs/changelog.md]: Maybe, we shouldn't do this.

Comment (gregorydlogan) [reply docs/guides/admin/docs/changelog.md]: This file has (traditionally) been auto generated. That being said, you're the rm here :)

Comment (Rillke) [reply docs/guides/admin/docs/changelog.md]: I used [auto-generation](https://github.com/opencast/helper-scripts/tree/master/release-management/create-changelog) (and GitHub told me I hit their rate limit) but removed the depadabot "Bump" PRs afterwards as anyone interested in them can go through the commit history. I'll probably add an auth option to the script after I verified it works correctly (i.e. is not endlessly sending requests) as unauthenticated users have stricter limitations.

Comment (gregorydlogan) [reply docs/guides/admin/docs/changelog.md]: I've hit those request limits, but mainly when I have to rerun the script. I just wait for an hour or two and it resets :D

File: docs/guides/admin/docs/releasenotes.md [hunk 2]
```diff
@@ -1,16 +1,76 @@
 # Opencast 12: Release Notes
 
-<div class=warn>
-Opencast 12 is not yet released!
-</div>
+Features
+--------
 
+- Opencast 12 ships Paella Player 6.5.6. Paella 7 supporting CSS customization
+  [can be configured as an alternative player](modules/paella.player7/configuration.md).
```
Comment (lkiesow) [inline docs/guides/admin/docs/releasenotes.md]:
```suggestion
- Opencast 12 ships Paella Player 6.5.6 as its new default player.
- Paella Player 7 supporting new features like CSS customization is now included and
  [can be configured as an alternative player](modules/paella.player7/configuration.md).
```

File: docs/guides/admin/docs/releasenotes.md [hunk 3]
```diff
@@ -1,16 +1,76 @@
...
+  Both parameters can be configured separately for read-only actions and those
+  that also update or delete, since arguably the success of the latter is more
+  important. Changing this config does not require a restart of Opencast. See
+  our [Elasticsearch docs](configuration/elasticsearch.md) for more details.
+- The Series Service does not require a Solr Index anymore, simplifying the
+  installation of Opencast.
+- The Workflow Service does not require a Solr Index anymore, simplifying the
+  installation of Opencast.
+- Workflows' data is now atomically stored in the database instead of XML.
+  [[#3376](https://github.com/opencast/opencast/pull/3376)]
```
Comment (lkiesow) [inline docs/guides/admin/docs/releasenotes.md]:
```suggestion
  [[#3376](https://github.com/opencast/opencast/pull/3376)] to improve access speed.
```

File: docs/guides/admin/docs/releasenotes.md [hunk 4]
```diff
@@ -1,16 +1,76 @@
...
+- You can now specify a fallback language and use a placeholder in the [Speech to Text Workflow Operation Handler](workflowoperationhandlers/speech-to-text-woh.md).
```
Comment (lkiesow) [inline docs/guides/admin/docs/releasenotes.md]:
```suggestion
- You can now specify a fallback language and use a placeholder in the Vosk-baased [Speech to Text Workflow Operation Handler](workflowoperationhandlers/speech-to-text-woh.md).
```

File: docs/guides/admin/docs/releasenotes.md [hunk 5]
```diff
@@ -1,16 +1,76 @@
...
+- Improved performance when rebuilding the Elastic Search Event Index. [[#3775](https://github.com/opencast/opencast/pull/3775)]
```
Comment (lkiesow) [inline docs/guides/admin/docs/releasenotes.md]:
```suggestion
- Improved performance when rebuilding the Elasticsearch event index. [[#3775](https://github.com/opencast/opencast/pull/3775)]
```

**Pull request merged**